ui.apps/src/main/content/jcr_root/apps/etoolbox-rollout-manager/clientlibs/rollout-manager-ui/js/console-ui.dialog.js

```diff
@@ -99,9 +101,19 @@
             .appendTo(item);
     }
 
+    function renderIcon() {
+        return new Coral.Icon().set({ icon: 'checkmark' });
+    }
+
+    function renderErrorIcon() {
+        return new Coral.Icon().set({ icon: 'close' });
+    }
+
     function createLogItem(message) {
-        const $checkbox = $(`<coral-checkbox class="rollout-log-item" value="${message}">`).text(message);
-        return $('<li>').append($checkbox);
+        const $item = $('<li class="rollout-log-item">').text(message);
+        const $span = $('<span class="rollout-icon-status">');
```

Comment:
Minor: 
Do we need this span? We can set class and change the icon of the coral item instead